.github/workflows/build.yaml

```
@@ -9,10 +9,11 @@ on:
 
 jobs:
   build:
-    name: 'PHP ${{ matrix.php.major }}.${{ matrix.php.minor }}'
-    runs-on: ubuntu-20.04
+    name: 'PHP ${{ matrix.php.major }}.${{ matrix.php.minor }} on ${{ matrix.operating-system }}'
+    runs-on: ${{ matrix.operating-system }}
     strategy:
       matrix:
+        operating-system: [ubuntu-latest, macOS-latest]
```

Please use itemized lists:

```suggestion
        operating-system:
          - ubuntu-latest
          - macOS-latest
```